Formula/l/llvm.rb

```diff
@@ -49,10 +60,39 @@ class Llvm < Formula
   # Fails at building LLDB
   fails_with gcc: "5"
 
+  # Support simplified triples in version config files.
+  # https://github.com/llvm/llvm-project/pull/111387
+  patch do
+    url "https://github.com/llvm/llvm-project/commit/88dd0d33147a7f46a3c9df4aed28ad4e47ef597c.patch?full_index=1"
+    sha256 "0acaa80042055ad194306abb9843a94da24f53ee2bb819583d624391a6329b90"
+  end
+
+  # Fix triple config loading for clang-cl
+  # https://github.com/llvm/llvm-project/pull/111397
+  patch do
+    url "https://github.com/llvm/llvm-project/commit/a3e8b860788934d7cc1489f850f00dcfd9d8b595.patch?full_index=1"
+    sha256 "6d8403fec7be55004e94de90b074c2c166811903ad4921fd76274498c5a60a23"
+  end
+
   def python3
     "python3.13"
   end
 
+  def clang_config_file_dir
+    etc/"clang"
+  end
+
+  # TODO: Should this be in brew?
+  def macos_kernel_version_map
+    {
+      11 => 20,
+      12 => 21,
+      13 => 22,
+      14 => 23,
+      15 => 24,
+    }
```

Comment:
Isn't this `MacOS.version.major.to_i => OS.kernel_version.major.to_i`

Reply:
Ah. I guess this needs all combinations. Probably makes sense to add to brew then.